allennlp/semparse/domain_languages/domain_language.py

```diff
@@ -313,8 +350,9 @@ def add_predicate(self, name: str, function: Callable):
         return_type = signature.return_annotation
         # TODO(mattg): this might need to just call PredicateType.get_type, or something - what if
         # one of these is a function?
```

Comment:
This TODO is done, right? The code below calls ``PredicateType.get_type``

Reply:
Thanks for the catch, fixed.